incubator/hnc/hack/ci-test.sh

```diff
@@ -14,6 +14,14 @@
 
 set -e
 
+# Not included or existing by default in Prow
+#  - This is part of an ongoing discussion present on the issue:
+#    https://github.com/kubernetes/test-infra/issues/9469,
+#  - In Prow, the GOPATH is set to /home/prow/go, whereas in
+#    the Docker container is /go, which is the default one.
```

Comment:
Maybe add the fact that other K8s projects like `controller-runtime` do the same thing.

Reply:
Noice. Will do :)